checks.d/http_check.py

```diff
@@ -143,6 +144,23 @@ class HTTPCheck(NetworkCheck):
 
     def __init__(self, name, init_config, agentConfig, instances):
         self.ca_certs = init_config.get('ca_certs', get_ca_certs_path())
+        proxy_settings = get_proxy(agentConfig)
+        if not proxy_settings:
+            self.proxies = None
+        else:
+            uri = "{host}:{port}".format(
```

Comment:
AFAIK it is compatible, the incompatible format is the one with the empty braces (`{}`).